.github/workflows/autofmtpr.yml

```diff
@@ -13,6 +13,8 @@ jobs:
         run: ./bin/hermit env -r >> "$GITHUB_ENV"
       - name: Auto format
         run: autofmt
+      - name: Build JVM
+        run: just build-jvm
```

Comment:
Shouldn't this be in the autofmt script?

Reply:
I will change it, it is potentially considerably slower than the other formatters though